misc/python/materialize/cli/mzconduct.py

```diff
@@ -946,27 +947,115 @@ def run(self, comp: Composition, workflow: Workflow) -> None:
             raise Failed(f"Unable to kill container {container_id}: {e}")
 
 
-@Steps.register("chaos-delay-docker")
-class ChaosDelayDockerStep(WorkflowStep):
-    """Delay the incoming and outgoing network traffic for a Docker service.
+class ChaosNetemStep(WorkflowStep):
```

Comment:
Might be worth mentioning in the docs that `netem` is the pumba network emulation suite of tests.